jsapp/xlform/src/view.surveyApp.coffee

```diff
@@ -734,13 +734,13 @@ module.exports = do ->
       $header = $et.closest('.card__header')
       card_hover_text = do ->
         if buttonName is 'settings'
-          t("[button triggers] Settings")
+          "[button triggers] " + t("Settings")
         else if buttonName is 'delete'
-          t("[button triggers] Delete Question")
+          "[button triggers] " + t("Delete Question")
         else if buttonName is 'duplicate'
-          t("[button triggers] Duplicate Question")
+          "[button triggers] " + t("Duplicate Question")
         else if buttonName is 'add-to-library'
-          t("[button triggers] Add Question to Library")
+          "[button triggers] " + t("Add Question to Library")
```

Comment:
FYI: I talked with John to verify if my understanding of the ancient knowledge is accurate.

Please simply remove these `[button triggers]` from the strings, and not add them back in any form. Most likely these were added back in the days as a way to suggest the context of these strings to translators. That function is long forgotten and there are better ways to provide context out there :)